src/lib.rs

```diff
@@ -223,7 +223,7 @@ pub fn main() -> Result<(), Error> {
                                                 references.get_sketch_slice(j, k),
                                                 references.sketch_size,
                                             );
-                                            dist = if *ani { ani_pois(dist, k_f32) } else { dist };
+                                            dist = if *ani { ani_pois(dist, k_f32) } else { 1 as f32 - dist };
```

Comment:
I think a bit neater as:
```suggestion
                                            dist = if *ani { ani_pois(dist, k_f32) } else { 1.0_f32 as f32 - dist };
```